src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter(person => person.sex === 'm');
+
+  men = century
+    ? men.filter(person => Math.ceil(person.died / 100) === century)
+    : men;
```

Comment:
```suggestion
  let men = people.filter(person => person.sex === 'm');

  men = century
    ? men.filter(person => Math.ceil(person.died / 100) === century)
    : men;
```
Стоп стоп, це якось дивно, фільтрувати одну змінну аж двома фільтрами. Давай придумаємо як об'єднати ці фільтри, і вже всередині нашого єдиного фільтру зробити перевірку, чи вказаний century, чи ні. В залежності від того повертати одні чи інші елементи